docs/using-bi-for-auth.md

```diff
@@ -75,16 +75,39 @@ Use the URL above from a browser, and watch how the authentication process unfol
 
 ### How to craft your Token URL
 
-Calling the token endpoint is the second step in the authorization flow. You will need to call the authorization endpoint first to retrieve an authorization code. You can find the base token URL under your application.
+Calling the token endpoint is the second step in the authorization flow. You will need to call the authorization endpoint first to retrieve an authorization code. You can find the base token URL under your application. 
 
-![Screenshot](./screenshots/TokenUrl.jpg)]
+![Screenshot](./screenshots/TokenUrl.jpg)
 
-The following request parameters can be POSTed as a part of the URL-encoded form values to the API:
+You'll also need to make note of your application's Token Endpoint Auth Method.
 
-```bash
-grant_type=authorization_code&
+![Screenshot](./screenshots/TokenAuthMethod.png)
+
+Client Secret Basic:  
+The client_id and client_secret are sent in the Basic Authorization header.
+
+```
+curl -X POST https://your-token-endpoint \
+--header 'Authorization: Basic {base64(<client_id>:<client_secret>)}' \
+--header "Content-Type: application/x-www-form-urlencoded" \
+--data-raw "grant_type=authorization_code
 &code=<code_return_from_authorization_response>
-&client_id=<client_id_from_application>
 &code_verifier=<code_verifier_from_PKCE_code_challenge_if_used_in_authorization_request>
-&redirect_uri=<redirect_uri_must_match_value_used_in_authorization_request>
-```
\ No newline at end of file
+&redirect_uri=<redirect_uri_must_match_value_used_in_authorization_request>"
+```
+
+Client Secret Post:  
+The client_id and client_secret are sent in the body of the POST request as a form parameter.
+
+```
```

Comment:
Let's add `bash` syntax highlighting to these.

Reply:
updated!